data/catalogs/deltares_data.yml

```diff
@@ -1510,19 +1636,23 @@ rivers_lin2019_v1:
     paper_ref: Lin et al. (2019)
     source_license: CC-BY-NC 4.0
     source_url: https://zenodo.org/record/3552776#.YVbOrppByUk
-    source_version: 1
+    processing_notes: hydrography/rivers_lin2019/README
```

Comment:
Did we do any more format than fileformat conversion? Important to know for dd reprocibility for external wflow users